app/Entities/Controllers/PageController.php

```diff
@@ -259,7 +260,9 @@ public function showDelete(string $bookSlug, string $pageSlug)
         $page = $this->pageRepo->getBySlug($bookSlug, $pageSlug);
         $this->checkOwnablePermission('page-delete', $page);
         $this->setPageTitle(trans('entities.pages_delete_named', ['pageName' => $page->getShortName()]));
-        $usedAsTemplate = Book::query()->where('default_template_id', '=', $page->id)->count() > 0;
+        $usedAsTemplate = 
+            Book::query()->where('default_template_id', '=', $page->id)->count() > 0 ||
+            Chapter::query()->where('default_template_id', '=', $page->id)->count() > 0;
```

Comment:
So if I'm getting you right you suggest to update the translation in this case, so that it includes chapters as well? Something like this: `'pages_delete_warning_template' => 'This page is in active use as a book or chapter default page template. These books or chapters will no longer have a default page template assigned after this page is deleted.',` ?

Reply:
Yeah, just like that

Reply:
ok done ;)